tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  mode: "jit",
+  purge: ["./pages/**/*.{js,ts,jsx,tsx}"],
```

Comment:
```suggestion
  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
```
I think you need to add the `components` folder here because we will be using it there as well.

Please refer to [my configuration here](https://github.com/zainfathoni/rumahberbagi-app/blob/d944ebbf972d528bc4728e77b96cb8abce801d74/tailwind.config.js#L4-L4) as a reference, thanks!